Dockerfile

```
@@ -6,7 +6,7 @@ RUN gradle jpi
 
 
 # Stage: Prod
-FROM jenkins/jenkins:2.176.1 as prod
+FROM jenkins/jenkins:2.176.4 as prod
```

Might as well use the latest:
```suggestion
FROM jenkins/jenkins:2.222.3 as prod
```

---

Cool. Are we bumping to the latest?